.github/workflows/ci.yaml

```diff
@@ -32,13 +32,38 @@ jobs:
           BUF_BREAKING_PROTO_INPUT: 'https://github.com/bufbuild/buf.git#branch=master,subdir=proto'
           BUF_INPUT_HTTPS_USERNAME: ${{ github.actor }}
           BUF_INPUT_HTTPS_PASSWORD: ${{ github.token }}
-      - name: docker
-        if: success()
-        run: bash .github/scripts/docker.bash
-        env:
-          DOCKER_BUILD_MAKE_TARGET: dockerbuildbuf
-          DOCKER_IMAGE: bufbuild/buf
-          DOCKER_USERNAME: ${{ secrets.DOCKER_USERNAME }}
-          DOCKER_TOKEN: ${{ secrets.DOCKER_TOKEN }}
-          DOCKER_LATEST_BRANCH: master
-          DOCKER_VERSION_TAG_PREFIX: v
+  docker:
```

Comment:
I think we've lost the check to ensure this isn't run on forks?

Reply:
https://github.com/bufbuild/buf/blob/master/.github/scripts/docker.bash#L20

Reply:
I will update the `if` conditional to `github.head_ref == "" && (github.ref == 'refs/heads/master' || contains(github.ref, 'refs/tags/v'))`

Reply:
I think the existing logic was actually incorrect. https://docs.github.com/en/actions/reference/context-and-expression-syntax-for-github-actions#github-context

`github.head_ref` is only set during a PR, so a push to master / a version tag in a fork should pass this conditional (which we do not want). I believe the correct check (for skipping forks; the master / tags/v checks should remain) is 
```
github.repository == 'bufbuild/buf'
```

Additionally, can you add a comment with some of the details here so that a reader unexperienced with github actions understands the intent.

Reply:
I think you are right. I am not too sure what would be the output of `github.head_ref` actually. I was just using the logic from https://github.com/bufbuild/buf/blob/master/.github/scripts/docker.bash#L20 directly.